src/functions.js

```
@@ -2680,24 +3099,37 @@ export function shrineBonusActive() {
-		mult: 1,
-		add: 0
+        mult: 1,
+        add: 0,
+        active: false
 	};
 
 	if (shrineBonusActive()){
 		switch(type){
 			case 'metal':
-				shrine_bonus.mult += +(global.city.shrine.metal / 100 * traits.magnificent.vars()[3]);
+                let metal = global.city.shrine.metal;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)